### PR TITLE
Service register

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/controllers/clients/notes_controller.rb [hunk 1]
```diff
@@ -6,10 +6,11 @@
 
 module Clients
   class NotesController < ApplicationController
+    include PjaxModalController
     include ClientPathGenerator
     include ClientDependentControllers
 
-    before_action :require_can_edit_window_client_notes_or_own_window_client_notes!
+    before_action :require_can_edit_window_client_notes_or_own_window_client_notes!, only: [:index, :create, :destroy]
```
Comment (awisema) [inline app/controllers/clients/notes_controller.rb:13]: Should there be a guard on alerts, or can anybody with a login see them? If anybody can see them, should there be a note on the form reminding people not to include confidential information?

File: app/controllers/cohorts/client_notes_controller.rb [hunk 2]
```diff
@@ -53,6 +53,7 @@ def set_cohort_and_client
     def note_params
       params.require(:grda_warehouse_client_notes_cohort_note).permit(
         :note,
+        :alert_active,
```
Comment (awisema) [inline app/controllers/cohorts/client_notes_controller.rb:56]: I think this was removed?

File: app/views/clients/notes/_note.haml [hunk 3]
```diff
@@ -1,10 +1,12 @@
+- show_destroy = if defined?(show_destroy) then show_destroy else true end
```
Comment (awisema) [inline app/views/clients/notes/_note.haml]: Maybe a comment to call out that showing the delete button is the default? Seems different that most of the code...

File: drivers/service_scanning/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## ServiceScanning README
```
Comment (awisema) [inline drivers/service_scanning/README.md:1]: Remove or replace?

File: app/models/grda_warehouse/data_source.rb [hunk 5]
```diff
@@ -84,6 +85,10 @@ class GrdaWarehouse::DataSource < GrdaWarehouseBase
     where(authoritative: true)
   end
 
+  scope :scannable, -> do
```
Comment (awisema) [inline app/models/grda_warehouse/data_source.rb:89]: Did you intend to move it this to `data_source_extension.rb`?

**Review by awisema — CHANGES_REQUESTED** [review]

File: app/models/grda_warehouse/data_source.rb (hunk 5 above)
Comment (eanders) [reply app/models/grda_warehouse/data_source.rb:89]: I didn't, but I could never get it to work there.

**Pull request merged**